### DISCUSSION TITLE
better civetman

### DISCUSSION THREAD

**Discussion by iacore** [DanielXMoore/Civet#1289] [General]
i don't know i'm doing.

at least i got it cleaned up.

https://github.com/iacore/civetman/

y'all can install this with `pnpm -g i github:iacore/civetman`

**Comment by edemaine** [comment]
Perhaps it's worth submitting some of these changes as a PR, to avoid unnecessary forking? (I'm not sure why some files were deleted... perhaps they were moved...)

**Reply by iacore** [reply]
i couldn't figure out how to make the package installable from git other, so i took the dumb approach.

i sent the PR. https://github.com/arstnei0/civetman/pull/3
